service/collector/chart_resource.go

```diff
@@ -36,12 +40,21 @@ var (
 		},
 		nil,
 	)
+
+	cordonExpireTimeDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(Namespace, "", "cordon_expire_time"),
+		"A metric of the expire time of cordoned chartconfig as unix seconds.",
+		[]string{
+			chartNameLabel,
```

Comment:
`chartName` would be enough, right?

Reply:
Check the `aws-operator` for instance. We could call this here `labelChart`. Additionally tenant cluster ID and organisation would maybe be useful for oncall and ops.

Reply:
I saw that cluster_id was added automatically after scraping so we don't need to add it, although I'm not sure what oraganization value would be. 
```chart_operator_chartconfig_status{
app="chart-operator",
cluster_id="tn6n3",
cluster_type="guest",
exported_namespace="giantswarm",
instance="master.tn6n3:443"
}```

Reply:
I see. My thinking was that something like the organization is dispatched via the CR labels.